src/conteiners/MovieListContainer.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux';
+import MovieList from '../components/MovieList/MovieList';
+import getFilms from '../modules/actions/getFilms';
+import { getListFilms, getGenre } from '../modules/selectors/index';
+
+const mapStateToPops = (state) => {
```

Comment:
for now we can use redux hooks like `useSelector `and `useDispatch`

Reply:
I use useSelector and useDispatch